xml/System.Text/Rune.xml

```diff
@@ -493,10 +493,18 @@ This method is very similar to <xref:System.Text.Rune.DecodeFromUtf8(System.Read
         <Parameter Name="value" Type="System.Text.Rune" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="value">The <see cref="T:System.Text.Rune" /> for which to get the numeric value.</param>
+        <summary>Gets the numeric value associated with the specified <see cref="T:System.Text.Rune" />.</summary>
+        <returns>The numeric value associated with the specified <see cref="T:System.Text.Rune" />, or -1 if the specified <see cref="T:System.Text.Rune" /> does not represent a numeric character.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This method has behavior equivalent to <xref:System.Globalization.CharUnicodeInfo.GetNumericValue>. Refer to that documentation for more information.
```

Comment:
This might be needed in order to explicitly point out that we are referring to a method form a different class.
```suggestion
This method has behavior equivalent to <xref:System.Globalization.CharUnicodeInfo.GetNumericValue?displayProperty=nameWithType>. Refer to that documentation for more information.
```